### PR TITLE
Fix/upgrade docker ce

### PR DESCRIPTION
**Pull request by fpetkovski**

### CODE REVIEW

**Review by boynux — APPROVED** [review]
SGTM

File: build/setup.d/20-install-packages.sh [hunk 1]
```diff
@@ -30,4 +30,4 @@ libswitch-perl
 
 echo "Installing packages..."
 
-apt-get install -y -q --no-install-recommends -o Dpkg::Options::="--force-confold" $pkgs >>install.log
+apt-get update -y && apt-get install -y -q --no-install-recommends -o Dpkg::Options::="--force-confold" $pkgs >>install.log
```
Comment (mseiwald) [inline build/setup.d/20-install-packages.sh:33]: Could you add a comment to explain why we need the apt options?

Comment (fpetkovski) [reply build/setup.d/20-install-packages.sh:33]: I am not entirely sure myself either. I thing the options were simply inherited from the Zalando repository: https://github.com/zalando-stups/taupage/blob/master/build/setup.d/20-install-packages.sh#L45

I simply added the apt-get update :)

Comment (mseiwald) [reply build/setup.d/20-install-packages.sh:33]: Ok nevermind :)

**Pull request merged**